squarify/__init__.py

```diff
@@ -167,11 +166,13 @@ def normalize_sizes(sizes, dx, dy):
     list[numeric]
         The normalized values.
     """
-    total_size = sum(sizes)
+    if isinstance(sizes, list):
+        import numpy as np
```

Comment:
Can you reimpl this without Numpy? It's not strictly a requirement for squarify, and I don't want to introduce it as one.